src/site/content/en/blog/vitals-field-measurement-best-practices/index.md

```diff
@@ -0,0 +1,276 @@
+---
```

Comment:
I'm realizing now that it's a bit funny that our main URL is `web.dev/vitals` whereas the main branding when we talk about this initiative is `Web Vitals`. I guess there's no action here, just noting that my first impression was that the URL should be `…/web-vitals-field-measurement-best-practices`.

Reply:
Yeah, Harleen and I discussed this and we thought it might be redundant to have web.dev/web-vitals ("web" twice). And this URL was just following that pattern.

Might not be a bad idea to have `/web-vitals` redirect to `/vitals` though...

Reply:
Added the redirect.